src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos ? todos.map(item => (
+      <TodoInfo
+        key={item.id}
+        todo={item}
+      />
+    )) : null}
```

Comment:
Consider using a more descriptive placeholder than just 'null' when there are no todos. It can be a user-friendly message informing that there are no TODO items.